packages/engine/errors/1014.md

```diff
@@ -0,0 +1,24 @@
+---
+original: "A rest parameter must be last in a parameter list."
+excerpt: "Rest parameter should be last"
+---
+
+Rest parameters tell me to group all remaining variables together, like this:
+
+```ts
+function doStuff(fruit: string, ...numbers: number[]){
+	// stuff
+}
+
+doStuff('banana', 1, 2, 3)
+```
+
+I get confused if they're not at the very end of the list, so please don't do something like this:
```

Comment:
```suggestion
I get confused if they're not at the very end of the list, because I won't know how many to group together. I'll find this confusing:
```